pkg/apiserver/debugapi/param_model_def.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debugapi
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/thoas/go-funk"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/debugapi/endpoint"
+)
+
+var APIParamModelText = endpoint.NewAPIParamModel("text")
+
+var APIParamModelEscapeText = APIParamModelText.Copy().Use(func(p *endpoint.ModelParam, ctx *endpoint.Context) {
+	p.SetValue(url.QueryEscape(p.Value()))
+	ctx.Next()
+})
+
+var falselyValues = []string{"false", "0", "null", "undefined", ""}
+
+var APIParamModelBool = endpoint.NewAPIParamModel("bool").Use(func(p *endpoint.ModelParam, ctx *endpoint.Context) {
+	if funk.Contains(falselyValues, p.Value()) {
+		p.SetValue("false")
+	} else {
+		p.SetValue("true")
+	}
+	ctx.Next()
+})
+
+var APIParamModelTags = endpoint.NewAPIParamModel("tags").Use(func(p *endpoint.ModelParam, ctx *endpoint.Context) {
```

Comment:
Seems that this is more like a MultiValue? (and maybe better to allow customizing the separator).